test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Caching/CachingTestContext.cs

```diff
@@ -314,25 +312,11 @@ public void WritePackagesConfig(PackageIdentity packageIdentity)
             File.WriteAllText(PackagesConfigPath, content);
         }
 
-        public void WriteProjectJson(PackageIdentity packageIdentity)
+        public void WriteProject(PackageIdentity packageIdentity)
         {
-            var content = $@"{{
-  ""dependencies"": {{
-    ""{packageIdentity.Id}"": ""{packageIdentity.Version}""
-  }},
-  ""frameworks"": {{
-    ""{PackageFramework.GetShortFolderName()}"": {{}}
-  }}
-}}";
-
-            File.WriteAllText(ProjectJsonPath, content);
-        }
-
-        public void WriteProject()
-        {
-            var content = Util.GetCSProjXML("project");
-
-            File.WriteAllText(ProjectPath, content);
+            var projectContext = SimpleTestProjectContext.CreateLegacyPackageReference("project", TestDirectory, FrameworkConstants.CommonFrameworks.Net472);
```

Comment:
Should this continue to use `PackageFramework` for consistency? (and should that be updated to Net472?)

Reply:
I think just bringing pack the  PackageFramework is the correct thing.
I honestly just missed it since I went through multiple iterations of this PR.

I don't think we need to update the framework since there's no build element to this test, so the exact framework is not relevant.